### PR TITLE
doc: add FAQ about redirect http To https

### PR DESCRIPTION
**Pull request by rhubard**

### CODE REVIEW

**Comment by membphis** [comment]
@rhubard https://github.com/apache/incubator-apisix/pull/1389#issuecomment-609541401

File: FAQ.md [hunk 1]
```diff
@@ -107,11 +107,30 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/2 -H 'X-API-KEY: edd1c9f034335
...
+
+here is the way:
+
+```shell
+curl -i http://127.0.0.1:9080/apisix/admin/global_rules/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "plugins": {
+        "serverless-pre-function": {
+          "phase": "rewrite",
+          "functions": ["return function() if ngx.var.scheme == 'http' and ngx.var.host == 'iresty.com' then ngx.header['Location'] = 'https://iresty.com' .. ngx.var.request_uri; ngx.exit(ngx.HTTP_MOVED_PERMANENTLY); end; end"]
```
Comment (moonming) [inline FAQ.md:122]: This command will throw an error, have you ever run it locally?

Comment (rhubard) [reply FAQ.md:122]: I run successfully in yaml mode

```yaml
global_rules:
  - plugins:
      serverless-pre-function:
        phase: "rewrite"
        functions:
          - "return function() if ngx.var.scheme == 'http' and ngx.var.host == 'xxx.com' then ngx.header['Location'] = 'https://xxx.com' .. ngx.var.request_uri; ngx.exit(ngx.HTTP_MOVED_PERMANENTLY); end; end"
```

**Comment by moonming** [comment]
you can try the `curl` command, which is different to yaml.

Thanks,
Ming Wen
Twitter: _WenMing


JunQing Hou <notifications@github.com> 于2020年4月19日周日 上午9:02写道：

File: FAQ_CN.md [hunk 2]
```diff
@@ -109,5 +110,23 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/2 -H 'X-API-KEY: edd1c9f034335
 }'
 ```
 
+## 如何通过APISIX支持http自动跳转到https？
```
Comment (juzhiyuan) [inline FAQ_CN.md:113]: Please add spaces on both sides of APISIX/http/https

File: FAQ.md [hunk 3]
```diff
@@ -107,11 +107,30 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/2 -H 'X-API-KEY: edd1c9f034335
 }'
 ```
 
+## How to redirect http To https via APISIX?
```
Comment (juzhiyuan) [inline FAQ.md:110]: `To` should be `to` in lowercase.

**Comment by moonming** [comment]
ping @rhubard

**Comment by rhubard** [comment]
> ping @rhubard

So sorry，I am on a business trip, I don’t have enough time to verify the above issues now.

**Comment by moonming** [comment]
it is ok, i will fix it, thx for your PR

JunQing Hou <notifications@github.com> 于 2020年4月21日周二 下午5:51写道：

**Pull request closed without merging**